### PR TITLE
Azure Migrate UI fixes

### PR DESCRIPTION
**Pull request by akn9050**

### CODE REVIEW

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md [hunk 1]
```diff
@@ -328,43 +328,39 @@ In this task, you will use Azure Migrate to create a migration assessment for th
 
 1. Continuing from Task 3, select **Assess** under **Azure Migrate: Server Assessment** to start a new migration assessment.
 
-    ![Screenshot of the Azure Migrate portal blade, with the '+Assess' button highlighted.](images/Exercise1/start-assess.png "Start assessment")
+    ![Screenshot of the Azure Migrate portal blade, with the '+Assess' button highlighted.](images/Exercise1/start-assess-v2.png "Start assessment")
 
-2. On the Assess servers blade, enter **SmartHotelAssessment** as the assessment name.
+2. On the **Create Assessment Basics** blade, ensure the Assessment type to be Azure VM and Discovery Source to be Machines discovered from Azure Migrate Appliance. Under **Assessment properties**, select **Edit..**
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - Line-of-business application migration.md:333]: Please remove extra period after **Edit..**

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md [hunk 2]
```diff
@@ -328,43 +328,39 @@ In this task, you will use Azure Migrate to create a migration assessment for th
...
 
     **Note:** There is no need to include the **smarthotelSQL1** or **AzureMigrateAppliance** VMs in the assessment, since they will not be migrated to Azure. (The SQL Server will be migrated to the SQL Database service and the Azure Migrate Appliance is only used for migration assessment.)
 
-6. Select **Next**, followed by **Create assessment**. On the **Azure Migrate - Servers** blade, select **Refresh** periodically until the number of assessments shown is **1**. This may take several minutes.
+6. Select **Next**, followed by **Create assessment**. On the **Azure Migrate - Windows, Linux and SQL Server** blade, select **Refresh** periodically until the number of assessments shown is **1**. This may take several minutes.
 
-    ![Screenshot from Azure Migrate showing the number of assessments as '1'.](images/Exercise1/assessments-refresh.png "Azure Migrate - Assessments (count)")
+    ![Screenshot from Azure Migrate showing the number of assessments as '1'.](images/Exercise1/ssessments-refresh-v2.png "Azure Migrate - Assessments (count)")
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - Line-of-business application migration.md:349]: Typo in the images URL, when I view the file, the image isn't broken so I'm not sure if it needs to be fixed, or if the typo also exists in the image file so it matches.

**Comment by DawnmarieDesJardins** [comment]
@akn9050 
Hi Amit.  Dwayne and I reviewed your PR, thank you.  Please see my comments above, I'm unable to edit your PR.  Once those two are addressed, assign the PR to me and I'll merge.  Have a great day!

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md (hunk 1 above)
Comment (akn9050) [reply Hands-on lab/HOL step-by step - Line-of-business application migration.md]: Hi Dawnmarie, I have removed the extra period, 

Thanks

File: Hands-on lab/HOL step-by step - Line-of-business application migration.md (hunk 2 above)
Comment (akn9050) [reply Hands-on lab/HOL step-by step - Line-of-business application migration.md]: It was a typo in image file as well, I have deleted the file and uploaded the image file again. Also, updated in HOL step-by step - Line-of-business application migration.md.

**Comment by akn9050** [comment]
> @akn9050
> Hi Amit. Dwayne and I reviewed your PR, thank you. Please see my comments above, I'm unable to edit your PR. Once those two are addressed, assign the PR to me and I'll merge. Have a great day!

Hi @DawnmarieDesJardins , I have made the changes as required, I am not getting option to assign the PR.

**Comment by DawnmarieDesJardins** [comment]
Much appreciated @akn9050!  Merging now and updating HTML docs.

**Pull request merged**